packages/create-hydrogen/src/prompts/init.ts

```diff
@@ -1,40 +1,38 @@
-import {ui, github, string} from '@shopify/cli-kit'
+import {ui, github, string, output} from '@shopify/cli-kit'
 
 const TEMPLATE_BASE = 'https://github.com/Shopify/hydrogen/templates/'
 const BRANCH = `dist`
-const TEMPLATE_DATA = {
+const TEMPLATE_NAME_DATA = {
   /* eslint-disable @typescript-eslint/naming-convention */
-  'demo-store-js': {
-    description: 'Demo Store',
+  'demo-store': {
+    name: 'Demo Store',
+    description: 'A simple store with a demo storefront',
   },
-  'demo-store-ts': {
-    description: 'Demo Store (TypeScript)',
+  'hello-world': {
+    name: 'Hello World',
+    description: 'A simple store with a hello world storefront',
   },
-  'hello-world-js': {
-    description: 'Hello World',
+}
+
+const LANGUAGE_NAME_DATA = {
+  js: {
+    name: 'JavaScript',
+    description: 'This is the default language, regular old JavaScript',
   },
-  'hello-world-ts': {
-    description: 'Hello World (TypeScript)',
+  ts: {
+    name: 'TypeScript',
+    description: 'A typed superset of JavaScript created by Microsoft',
```

Comment:
I think I would use different descriptions for these. @gfscott what do you think? 

Or not use any.